tools/ci_build/build.py

```diff
@@ -1822,8 +1822,12 @@ def build_nuget_package(source_dir, build_dir, configs, use_cuda, use_openvino,
             run_subprocess(cmd_args, cwd=csharp_build_dir)
 
         if is_windows():
-            # this path is setup by cmake/nuget_helpers.cmake for MSVC on Windows
-            nuget_exe = os.path.normpath(os.path.join(native_dir, config, "nuget_exe", "src", "nuget.exe"))
+            if use_openvino:
```

Comment:
provide more details on why this if block is needed for openvino?
why doesn't the else path work?

Reply:
The offline option to build nuget packages --build_nuget does not build a directory nuget_exe/src/nuget.exe... I suppose the directory is created through some other ci pipelines to build nuget packages on window. We have not explored to the build pipelines for now . Here is an explanation from Scott where he mentions why a directory is needed
"In our build pipeline, where this script is primarily used, we assembly native libraries from various platforms (linux, mac, mobile) in the nuget-artifacts directory and package those. I don’t know the history as to why we implicitly determine it’s that sort of build by the presence of the nuget-artifacts directory. That could probably be made more explicit and/or use a different directory for the artifacts dir to what the local build does so they don’t clash."

Reply:
thanks for the explanation!